internal/blockchain/fabric/fabric.go

```diff
@@ -897,6 +897,21 @@ func (f *Fabric) NormalizeContractLocation(ctx context.Context, ntype blockchain
 	return encodeContractLocation(ctx, ntype, parsed)
 }
 
+func (f *Fabric) StringifyContractLocation(ctx context.Context, location *fftypes.JSONAny) (string, error) {
```

Comment:
Can you help me understand the reason for this function?

Are we changing how we store the location in some way that means it's not a JSON object?

Reply:
(... came back to this in next comment)